packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.tsx

```diff
@@ -14,22 +21,39 @@ import {
 import { formatAddress } from "@utils/string-utils";
 import ThemeModeContainer from "@containers/theme-mode-container/ThemeModeContainer";
 import { AccountModel } from "@models/account/account-model";
+import IconPolygon from "../icons/IconPolygon";
+import IconFailed from "../icons/IconFailed";
+
+const URL_REDIRECT =
+  "https://gnoscan.io/accounts/g14qvahvnnllzwl9ehn3mkph248uapsehwgfe4pt";
```

Comment:
What does this url mean?

Reply:
Just Fake DATA (from log bug Kyungsu @harryoh )